packages/cubejs-schema-compiler/scaffolding/ScaffoldingTemplate.js

```diff
@@ -40,16 +40,16 @@ class ScaffoldingTemplate {
   schemaDescriptorForTable(tableSchema) {
     return {
       cube: tableSchema.cube,
-      sql: `SELECT * FROM ${tableSchema.schema}.${tableSchema.table}`, // TODO escape
+      sql: `SELECT * FROM "${tableSchema.schema}"."${tableSchema.table}"`, // TODO escape
       joins: tableSchema.joins.map(j => ({
         [j.cubeToJoin]: {
-          sql: `\${CUBE}.${j.thisTableColumn} = \${${j.cubeToJoin}}.${j.columnToJoin}`,
+          sql: `\${CUBE}."${j.thisTableColumn}" = \${${j.cubeToJoin}}."${j.columnToJoin}"`,
           relationship: j.relationship
         }
       })).reduce((a, b) => ({ ...a, ...b }), {}),
       measures: tableSchema.measures.map(m => ({
         [this.memberName(m)]: {
-          sql: m.name,
+          sql: `"${m.name}"`,
```

Comment:
Same here. Would be also great to check if we actually need to escape before escaping to keep schema clean.

Reply:
Updated,  please check if it works.